dashboard/lib/tasks/asset_sync.rake

```diff
@@ -0,0 +1,11 @@
+namespace :assets do
+  desc 'Synchronize assets to S3'
+  task :sync => :environment do
+    # AWS CLI implements an optimized `sync` utility without any Ruby SDK equivalent.
+    `aws s3 sync #{CDO.dashboard_dir}/public/assets s3://#{CDO.assets_bucket}/#{rack_env}/assets --acl public-read --cache-control 'max-age=31536000'`
```

Comment:
Just making sure I understand this - we are not blowing away the old assets when we do this upload because they are all fingerprinted, right?

Say daemon, frontendA and frontendB are all at version 1, using static asset application-1.js.  Now I want to deploy version 2, which contains static application-2.js.

1. daemon builds version 2.
2. daemon pushes application-2.js to s3://cdo-dist/production/application-2.js.  s3://cdo-dist/production/application-1.js is still intact.
3. daemon upgrades frontendA.  Now frontendA is pointing to application-2.js while frontendB is still pointing to application-1.js

And, thinking this through, this would work even if we had the apps build start doing its own fingerprinting instead of letting sprockets do it.

Okay, I don't have a question.  Just thinking out loud.